Wikipedia/Code/ReadingThemesControlsViewController.swift

```diff
@@ -69,7 +81,7 @@ open class ReadingThemesControlsViewController: UIViewController {
         button.borderWidth = 2
         button.isEnabled = false
         button.borderColor = theme.colors.link
-
+        button.accessibilityTraits = UIAccessibilityTraitSelected
```

Comment:
I think this needs a corresponding `button.accessibilityTraits = UIAccessibilityTraitButton` in `func removeBorderFrom(_ button: UIButton) {`